source/transfer_list.rst

```diff
@@ -84,12 +86,12 @@ version field determines the contents of the handoff start header.
    * - size
      - 0x4
      - 0x8
-     - The number of bytes occupied by the TL. This field accounts for the size of the TL header plus the size of all the entries contained in the TL. This field must be updated when any entry is added to the TL.
+     - The number of bytes occupied by the TL. This field accounts for the size of the TL header plus the size of all the entries contained in the TL. It must be a multiple of 16 (i.e. it counts the inter-TE padding after the end of the last TE). This field must be updated when any entry is added to the TL.
```

Comment:
nit: s/counts/includes/, although I'm not sure this change is needed at all depending on the result of other comments.

Reply:
I can update the wording. Regardless of the checksum discussion I think it's important that it is well-defined whether the padding is included in that size value or not, and with the current wording that's not super clear to me. I think including the padding is preferable because it makes it easier to add a new tag.

Reply:
OK, I'm happy to include the padding in the size and I agree it's not clear currently.

Reply:
Updated in latest push.